egi_notebooks_hub/d4science.py

```diff
@@ -175,6 +190,12 @@ class D4ScienceOauthenticator(GenericOAuthenticator):
         config=True,
         help="""The OIDC URL for D4science""",
     )
+    d4science_infosys_url = Unicode(
+        D4SCIENCE_INFOSYS_URL,
+        config=True,
+        help="""The URL for the Information System of D4science""",
```

Comment:
this is more the URL for the GenericResources for JupyterHub

Reply:
fixed